### PR TITLE
[PPML] Update README of occlum Spark K8S example 

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md [hunk 1]
```diff
@@ -24,5 +24,6 @@ bash build-docker-image.sh
 3. Modify `executor.yaml`
 
 ```
+kubectl apply -f executor.yaml
 ./run_pi.sh
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md:28]: run_spark_pi.sh

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md [hunk 2]
```diff
@@ -24,5 +24,6 @@ bash build-docker-image.sh
 3. Modify `executor.yaml`
 
 ```
+kubectl apply -f executor.yaml
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md:27]: Remove this line. We don't need to launch executor pod. Spark submit in run_spark-pi will handle this.

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md (hunk 1 above)
Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md]: Thanks, done~

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md (hunk 2 above)
Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md]: Thanks, done~

**Pull request merged**